Python/specialize.c

```diff
@@ -21,7 +21,7 @@
 extern const char *_PyUOpName(int index);
 
 /* For guidance on adding or extending families of instructions see
- * ./adaptive.md
+ * ../InternalDocs/interpreter.md `Specialization` section.
```

Comment:
I don't see the need for relative paths in comments. Why not just make it this?
```suggestion
 * InternalDocs/interpreter.md `Specialization` section.
```